### PR TITLE
System Stability

### PR DESCRIPTION
**Pull request by BenGibb**
Collects System Stability statistics from Win32_ReliabilityStabilityMetrics and returns:

- Minimum stability over the collected period
- Average stability over the collected period
- Maximum stability over the collected period
- The last generated Stability Index
- Date and time of the last generated Stability Index

### CODE REVIEW

File: PowerShell Scanners/System Stability/System Stability.ps1 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[CmdletBinding()]
+param (
+    # Number of System Stability Indexes to return (generated hourly)
+    [Int]$Count = (24 * 7 * 4) #-Four week rolling window
+)
+
+# Gather $Count worth of ReliabilityStabilityMetrics
+$StabilityMetrcs = Get-WmiObject -ClassName Win32_ReliabilityStabilityMetrics | Select-Object -First $Count
```
Comment (PDQDakota) [inline PowerShell Scanners/System Stability/System Stability.ps1]: Could you swap out `Get-WmiObject` for `Get-CimInstance`? The new Cim cmdlet will get the same info a little bit faster here.

**Review by PDQDakota — CHANGES_REQUESTED** [review]
Thanks for this, the reliability monitor is really neat!

File: PowerShell Scanners/System Stability/System Stability.ps1 (hunk 1 above)
Comment (BenGibb) [reply PowerShell Scanners/System Stability/System Stability.ps1]: I've made the changes on my end (also fixed/cleaned up the TimeGenerated as a side effect)
How do I get the changes to you?
(still rather green when it comes to Git)

Comment (PDQDakota) [reply PowerShell Scanners/System Stability/System Stability.ps1]: A push from the same branch (which you've already done!) will get the updates to us.

**Review by PDQDakota — APPROVED** [review]
Thank you for helping out!

**Review by Colby-PDQ — APPROVED** [review]

**Pull request merged**